addons/libs/packets/fields.lua

```diff
@@ -1554,15 +1554,19 @@ fields.incoming[0x026] = L{
     {ctype='data[22]',          label='_unknown2',          const=0},           -- 06
 }
 
--- Encumbrance Release
+local function mid(val)
```

Comment:
I assume this should stand for "message ID". I find the name a bit odd. Personally I'd just use a function like this:
```lua
local function sub(amount, val)
    return val - amount
end
```

Then you can use it like so:
```lua
    {ctype='unsigned short',    label='Message ID',         fn=sub+{0x8000}},   -- 0A
```

This is more general and we could use it in other places as well (there are several packets that contain values with constants offsets). You might also add an `add` function that does the same but adding the amount.

I would also place the function at the top, along with the others. Take a look at [line 65](https://github.com/Windower/Lua/blob/dev/addons/libs/packets/fields.lua#L65).